e2e/next/src/next.test.ts

```diff
@@ -123,7 +123,7 @@ describe('Next.js Applications', () => {
 
     // Building the app throws if dependencies haven't been built yet
     expect(() => {
-      runCLI(`build ${appName}`);
+      runCLI(`build ${appName} --buildLibsFromSource false`);
```

Comment:
Does it work without `--buildLibsFromSource false`?

Reply:
no, without that the build no longer throws - which is what this expectation asserts. in other words omitting `buildLibsFromSource` - passes the build without having built the buildable dependency libs